src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -39,7 +39,8 @@ object Boot extends App {
     val gcsDAO = new HttpGoogleCloudStorageDAO(
       gcsConfig.getString("secrets"),
       new FileDataStoreFactory(new File(gcsConfig.getString("dataStoreRoot"))),
-      gcsConfig.getString("redirectBaseURL")
+      gcsConfig.getString("redirectBaseURL"),
+      gcsConfig.getInt("deletedBucketCheckSeconds")
```

Comment:
I think hard-coding is fine for durations. We already do it everywhere with route timeouts.